toolchain/semantics/testdata/var/decl_assign.carbon

```diff
@@ -0,0 +1,37 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
Please rename this file to `decl_with_init.carbon` or something like that so that this case and `global_decl_with_init.carbon` are consistent.

Reply:
Oops, fixing to decl_with_init.